lib/translator.js

```diff
@@ -36,19 +38,37 @@ Translator.prototype.getLocales = function() {
   return this._locales;
 };
 
+var createPoParseFn = function(lang) {
+  return function(msgDir) {
+    var fp = path.join(msgDir, lang, 'LC_MESSAGES', 'messages.po');
+    var po = '';
+    try{
+      po = fs.readFileSync(fp, {encoding: 'utf8'});
+    }catch(err) {
+      if(err.code !== 'ENOENT') {
```

Comment:
otherwise every source directory containing `.po` files(i.e. `msgDir` in the above function) needs to have a `/<language>/LC_MESSAGES/messages.po` or `fs.readFileSync` throws a `ENOENT`. this try/catch allows you to include source directories with translations for only some of your supported languages without blowing up your translation task. maybe we don't care/want that though? I could be convinced to go either way.